assets/va-robots.txt

```diff
@@ -0,0 +1,21 @@
+User-agent: Synapse
+Disallow: /
+
+User-agent: Googlebot
+Disallow: /
+
+User-agent: Bingbot
+Disallow: /
+
+User-agent: Slurp
+Disallow: /
+
+User-agent: DuckDuckBot
+Disallow: /
+
+User-Agent: *
```

Comment:
I'm not search what search.gov needs to index this page- i assume disallowing * will prevent it from indexing preview.va.gov. 

Opted to specifically target the top 4 search engines- should make this easier to update when we launch as well.